pyproject.toml

```diff
@@ -15,15 +15,16 @@ requests = "^2.21"
 
 [tool.poetry.dev-dependencies]
 pytest = "^4.1"
-ipdb = "^0.11.0"
 flake8 = "^3.6"
 flake8-isort = "^2.6"
 pylint = "^2.2"
 mypy = "^0.660.0"
 requests-mock = "^1.5"
-libero-schemas = {git = "https://github.com/libero/schemas.git"}
 pytest-mock = "^1.10"
 pytest-socket = "^0.3.3"
+pdbpp = "^0.10.0"
```

Comment:
If I understand correctly, new debugger, in dev dependencies :+1:

Reply:
yes, when building the jats-ingester I found a new debugger which is lightweight, has less dependencies than `ipdb` and doesn't require any extra configuration to run.